### ISSUE TITLE
unable to find icon for domain threshold / resizeObserver loop completed with undelivered notifications

### ISSUE THREAD

**Issue by Mariusthvdb** [home-assistant/frontend#12726]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

is flooding the logs when on config/devices/dashboard (helpers) 

when on /config/entities, the 

only the `ResizeObserver loop completed with undelivered notifications`  is logged, and also causing the `frontend.js.latest.202205160` ERROR
### Describe the behavior you expected

.... 

### Steps to reproduce the issue

1. go to config/devices/dashboard (helpers) 
2.
3.
...

### What version of Home Assistant Core has the issue?

2022.6.0.dev20220518

### In which browser are you experiencing the issue with?

any?

### Which operating system are you using to run this browser?

macOS  Monterey 12.3.1

### Javascript errors shown in your browser console/inspector

```txt
2022-05-18 23:12:22 ERROR (MainThread) [frontend.js.latest.202205160] :0:0 Script error.
```

### Additional information

<img width="702" alt="Schermafbeelding 2022-05-18 om 23 12 53" src="https://user-images.githubusercontent.com/33354141/169157043-0e071e48-dad1-4e87-904a-81ffa5bcfaa4.png">

<img width="825" alt="Schermafbeelding 2022-05-18 om 23 12 46" src="https://user-images.githubusercontent.com/33354141/169157073-b252b401-3079-4aec-bc8d-e72829ddae94.png">

![May-21-2022 12-48-05](https://user-images.githubusercontent.com/33354141/169648192-a994f1f4-222e-408e-81e7-4bc785a6428f.gif)

**Comment by Mariusthvdb** [comment]
just thought Id check in 2022.6.1: still happening

**Comment by Mariusthvdb** [comment]
and still in 2022.7

![Uploading Schermafbeelding 2022-06-28 om 15.05.22.png…]()

**Comment by Mariusthvdb** [comment]
just noticed this is still very much alive in 2022.9 (beta)

**Comment by steverep** [comment]
Is this different from #9022?

**Comment by steverep** [comment]
The "unable to find icon" error should have been resolved by #13024.

The `ResizeObserver` error is unrelated but can be tracked by #9022, so closing this one as done.
